public/docs/_examples/template-syntax/dart/lib/app_component.html

```diff
@@ -655,31 +654,31 @@ <h4 id="ngFor-trackBy">NgForTrackBy</h4>
 <!-- template local variable -->
 <hr><h2 id="local-vars">Template local variables</h2>
 
-<!-- #docregion var-phone -->
+<!-- #docregion ref-phone -->
 <!-- phone refers to the input element; pass its `value` to an event handler -->
 <input #phone placeholder="phone number">
 <button (click)="callPhone(phone.value)">Call</button>
 
 <!-- fax refers to the input element; pass its `value` to an event handler -->
-<input var-fax placeholder="phone number">
+<input ref-fax placeholder="fax number">
 <button (click)="callFax(fax.value)">Fax</button>
-<!-- #enddocregion var-phone -->
+  <!-- #enddocregion ref-phone -->
 
 <h4>Example Form</h4>
-<!-- #docregion var-form -->
-<!-- #docregion var-form-a -->
+<!-- #docregion ref-form -->
+<!-- #docregion ref-form-a -->
 <form (ngSubmit)="onSubmit(theForm)" #theForm="ngForm">
-<!-- #enddocregion var-form-a -->
+<!-- #enddocregion ref-form-a -->
   <div class="form-group">
     <label for="name">Name</label>
-    <input id="name" class="form-control" required ngControl="firstName"
-        [(ngModel)]="currentHero.firstName">
+    <input class="form-control" required ngControl="firstName"
+      [(ngModel)]="currentHero.firstName">
```

Comment:
Revert to old formatting here, prefer leading with 4 spaces when starting a new line with attribute or property bindings.